aws/v16.2.0/release.yaml

```
@@ -0,0 +1,73 @@
+# Generated with:
+# /home/whites/workspace/devctl/devctl release create --name 16.2.0 --provider aws --base 16.1.0 --app cert-exporter@2.0.0 --component app-operator@5.3.0 --component containerlinux@2983.2.1 --component etcd@3.4.18 --overwrite
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
+  creationTimestamp: null
+  name: v16.2.0
+spec:
+  apps:
+  - componentVersion: 1.4.0
```

The componentVersion also needs a bump
```suggestion
  - componentVersion: 1.5.0
```